src/DynamoRevit/DynamoRevit.csproj

```diff
@@ -57,8 +62,9 @@
       <HintPath>$(DYNAMOAPI)\DynamoServices.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Greg, Version=1.0.5900.25656, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Greg.1.0.5900.25656\lib\net45\Greg.dll</HintPath>
+    <Reference Include="Greg, Version=1.0.5959.26179, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\..\Program Files\Dynamo\Dynamo Core\1.2\Greg.dll</HintPath>
```

Comment:
Greg.dll needs to be referenced from packages folder because it gets pulled from NuGet package at build time.